npm/qsharp/src/compiler/compiler.ts

```diff
@@ -140,26 +140,18 @@ export class Compiler implements ICompiler {
 
   async getAst(
     code: string,
-    languageFeatures?: string[],
-    profile?: TargetProfile,
+    languageFeatures: string[],
+    profile: TargetProfile,
```

Comment:
Does making these non-optional have an affect on QCOM? I was under the impression that we did this to avoid breaking changes to the API surface here, but maybe I'm remembering that incorrectly (@minestarks, do you remember?).

Reply:
This is ok. This would have been breaking if `getAst` or `getHir` were used in QCOM, but they're not. I like this version better since it avoids adding a meaningless default.